### PR TITLE
Implement phase 2

### PR DESCRIPTION
**Pull request by JasonGlazer**
Adding major data groups related to HVAC in Phase 2 of development based on new EnergyPlus tabular output reports.

### CODE REVIEW

**Comment by JasonGlazer** [comment]
@Myoldmopar I have the script to a good point supporting a bunch of new data groups in the RMD file. Please take a look and let me know what you think. I still need to add unit tests for the new code, hopefully next week.

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ Additional warning messages may appear when including the monthly predefined rep
 This will create filename_out.json files when EnergyPlus is run at the command line. 
 
 Note: This utility was designed to work with files produced using EnergyPlus at the command line. Some file renaming might be necessary if using EP-Launch. 
-If using EP-Launch the eplusout.json and eplusout_hourly.json files may be found in the EPTEMP directory without the specific file name.
+If using EP-Launch, the eplusout.json and eplusout_hourly.json files may be found in the EPTEMP directory without the specific file name.
```
Comment (Myoldmopar) [inline README.md:86]: Any thought about the new Python EP-Launch?  If we merge the big EP-Launch PR down and do a full release of it, would that make it more appropriate to mention here?

File: README.md [hunk 2]
```diff
@@ -138,19 +138,78 @@ These enumerated lists are found here:
...
+ - Terminal
+ - Pump
+ - Boiler
+ - Chiller
+ - HeatRejection
+ - ExteriorLighting
+
+
+Please note that all data groups listed above still have some data elements that are not implemented and, in many cases, many data elements that are not implemented. The ones that 
+were implemented often because they were easy to implement and not necessarily based on the need for the data element.
```
Comment (Myoldmopar) [inline README.md:187]: There's nothing _wrong_ with saying this, but I'm not sure it's meaningful.

File: energyplus_rmd/test/test_translator.py [hunk 3]
```diff
@@ -1443,11 +4103,1633 @@ def test_add_serial_number_nested(self):
...
+                    'PERIMETER_TOP_ZN_3 ZN REHEAT COIL': {'plant_loop_name': 'HOT WATER LOOP'},
+                    'PERIMETER_TOP_ZN_4 ZN REHEAT COIL': {'plant_loop_name': 'HOT WATER LOOP'},
+                    'VAV_BOT WITH REHEAT CLG COIL': {'plant_loop_name': 'CHILLED WATER LOOP'},
+                    'VAV_BOT WITH REHEAT MAIN HTG COIL': {'plant_loop_name': 'HOT WATER LOOP'},
+                    'VAV_MID WITH REHEAT CLG COIL': {'plant_loop_name': 'CHILLED WATER LOOP'},
+                    'VAV_MID WITH REHEAT MAIN HTG COIL': {'plant_loop_name': 'HOT WATER LOOP'},
+                    'VAV_TOP WITH REHEAT CLG COIL': {'plant_loop_name': 'CHILLED WATER LOOP'},
+                    'VAV_TOP WITH REHEAT MAIN HTG COIL': {'plant_loop_name': 'HOT WATER LOOP'}}
+
+        self.assertEqual(gathered_coil_connections, expected)
```
Comment (Myoldmopar) [inline energyplus_rmd/test/test_translator.py:5735]: Wow, those are some exhaustive tests!

File: energyplus_rmd/test/test_validator.py [hunk 4]
```diff
@@ -16,7 +17,7 @@ def test_invalid_dict(self):
         v = Validator()
         passed, message = v.validate_rmd(test2)
         self.assertFalse(passed)
-        self.assertEqual(message, "invalid: 'id' is a required property")
+        self.assertEqual(message, "invalid: 'id' is a required property at $")
```
Comment (Myoldmopar) [inline energyplus_rmd/test/test_validator.py:20]: What's the $?

**Review by Myoldmopar — COMMENTED** [review]
Minor comments, but this is a huge effort that appears well tested and ready to go.

**Comment by Myoldmopar** [comment]
This is giving code coverage of 95%, which is really high and great.  I'm not sure what develop is at because the last commit was too long ago to see in the GHA logs.  If develop is already 100%, then it might be worth looking at those last few missing coverage spots (https://github.com/JasonGlazer/createRulesetModelDescription/actions/runs/7575402982/job/20631931543?pr=17#step:5:27) in this branch and see if just a tweak of a test or two could cover them meaningfully.  If not, no big deal.

File: README.md (hunk 1 above)
Comment (JasonGlazer) [reply README.md:86]: Sure, that would be a good idea.

File: energyplus_rmd/test/test_validator.py (hunk 4 above)
Comment (JasonGlazer) [reply energyplus_rmd/test/test_validator.py:20]: It is kind of weird but the validator message includes the dollar sign. It is "err.json_path" maybe it means an empty json_path.

**Comment by JasonGlazer** [comment]
> This is giving code coverage of 95%, which is really high and great. I'm not sure what develop is at because the last commit was too long ago to see in the GHA logs. If develop is already 100%, then it might be worth looking at those last few missing coverage spots (https://github.com/JasonGlazer/createRulesetModelDescription/actions/runs/7575402982/job/20631931543?pr=17#step:5:27) in this branch and see if just a tweak of a test or two could cover them meaningfully. If not, no big deal.

We were at 91% before. So I thought 95% was a good improvement.

oldCodeCoverage

**Comment by Myoldmopar** [comment]
OK, thanks @JasonGlazer.  I'm good with merging this in.  The other PRs don't need to go in first or anything, do they?

**Comment by JasonGlazer** [comment]
Merge away. I removed one of the pull requests that was already part of this branch. The other two I need to circle back on and update.

**Pull request merged**